spec/controllers/drivers_controller_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+require 'rspec/json_expectations'
```

Comment:
@domenic99 I think, it's better to move it to test or spec helper, as far as it will be required frequently.